deploy-agent/deployd/download/http_download_helper.py

```diff
@@ -25,7 +25,7 @@
 class HTTPDownloadHelper(DownloadHelper):
 
     def _download_files(self, local_full_fn):
-        download_cmd = ['curl', '-o', local_full_fn, '-ks', self._url]
+        download_cmd = ['curl', '-o', local_full_fn, '-fks', self._url]
```

Comment:
Please also bump the verison in setup.py

Reply:
Done